src/modules/esl-event-listener/core/targets/swipe.target.ts

```diff
@@ -28,6 +32,7 @@ export interface ESLSwipeGestureSetting {
  */
 export class ESLSwipeGestureTarget extends SyntheticEventTarget {
   protected static defaultConfig: Required<ESLSwipeGestureSetting> = {
+    skipOnScroll: true,
```

Comment:
by default we'll definitely skip scrolling content?

Reply:
That's what we agreed on